src/video.tsx

```diff
@@ -546,16 +547,16 @@ const VideoPlayer = forwardRef<VideoPlayerRef, VideoProps>(
           }
           if (x < leftDoubleTapBoundary) {
             doubleLeftOpacity.value = 1;
-            rippleLeft.current?.onPress({ x, y });
+            // rippleLeft.current?.onPress({ x, y });
             runOnJS(seekByStep)(true);
```

Comment:
I'm trying to do some work to figure this out too and this line and the corresponding rippleRight call at line 556 is the issue. We clearly aren't doing any actual assignment here, just assigning an `onPress` function in ripple.tsx on line 61. But something here is getting read as an assignment to the (non mutable) refObjects that that the ripple refs are created as. Working on trying to understand what could be going on here. My intuition is that this is less of a react-native-reanimated issue and more of a react/typescript issue. Any thoughts?